gatsby-site/src/components/team-member/team-member.css

```diff
@@ -0,0 +1,112 @@
+:root {
+  --colarkblue: #0111FC;
+  --colarklightblue: #E3E5FC;
+}
+
+.team-member {
+  position: relative;
+  display: inline-block;
+}
+
+img.team-member__photo {
+  width: 100px;
+  height: 100px;
+  border-radius: 50%;
+}
+
+.team-member__photo-div {
+  position: relative;
+  z-index: 1;
+}
+
+.alina-photo {
+  top: 200px;
+  left: 200px;
+}
+
+.ellie-photo {
+  top: 40px;
+  left: 240px;
+}
+
+.lian-photo {
+  bottom: 20px;
+  left: 150px;
+}
+
+.olga-photo {
+  top: 150px;
+  left: 520px;
+}
+
+.claire-photo {
+  top: 20px;
+  left: 300px;
+}
+
+.someone-photo {
+  top: 240px;
+  right: 330px;
+}
+
+.team-member__name, .team-member__title, .team-member__email, .team-member__linkedin {
+  display: none;
+}
+
+.tooltiptext {
+    display: none;
+    width: 120px;
+    max-height: 300px;
+    background-color: var(--colarkblue);
+    border: 1px solid var(--colarkblue);
+    color: #fff;
+    text-align: center;
+    border-radius: 6px;
+    padding: 10px;
+    position: absolute;
+    z-index: 3;
+    top: 105px;
+    left: -10px;
+    opacity: 0.9;
+    transition: opacity 0.3s;
+    font: 20px "Roboto", sans-serif;
+    font-weight: 100;
+}
+
+.tooltiptext--name {
+  font-weight: 400;
+}
+
+.tooltiptext--title {
+  font-size: 18px;
+  color: var(--colarklightblue);
+}
+
+.tooltiptext::before {
+    content: "";
+    position: absolute;
+    bottom: 100%;
+    left: 50%;
+    margin-left: -5px;
+    border-width: 5px;
+    border-style: solid;
+    border-color: transparent transparent var(--colarkblue) transparent;
+}
+
+div.alina-photo:hover .tooltiptext,
+div.alina-photo:active .tooltiptext,
+div.alina-photo .tooltiptext:hover,
+div.lian-photo:hover .tooltiptext,
+div.lian-photo:active .tooltiptext,
+div.lian-photo .tooltiptext:hover,
+div.ellie-photo:hover .tooltiptext,
+div.ellie-photo:active .tooltiptext,
+div.ellie-photo .tooltiptext:hover,
+div.olga-photo:hover .tooltiptext,
+div.olga-photo:active .tooltiptext,
+div.olga-photo .tooltiptext:hover,
+div.claire-photo:hover .tooltiptext,
+div.claire-photo:active .tooltiptext,
+div.claire-photo .tooltiptext:hover {
+    display: inline-block;
```

Comment:
is there a base class like "photo" that we could use so we don't have to have all these named declarations?

Reply:
I couldn't tell you why, but I tried that and it doesn't work for the tooltip. (I took z-indexes into account, too).  I did find that ".tooltoptext:hover" is redundant, since its part of the photo div anyway, so the block is a bit smaller now.

Reply:
Ahh!  It worked once I swapped the class names from "alina-photo team-member__photo-div " to "team-member__photo-div alina-photo"!!  It doesn't make sense!  But you got your wish.